lab/changelog.md

```diff
@@ -4,6 +4,56 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Lab v2.5.0 - <TODO: Add the release date>
+
+## What's New
+
+* In case of multiple Cypher queries in the query editor, you will be able to select a single query and run
+  just that one without a need to comment out all the other Cypher queries.
+
+<TODO: Add screenshot here>
+
+* A limit of having one query execution view has been removed and you can have multiple query executions now. It makes
+  life easier especially when you want to compare different Cypher queries side to side in terms of speed or
+  query results.
+
+<TODO: Add screenshot here>
+
+* Showing only the last result when running multiple Cypher queries at once has been changed. Now, results for each
+  Cypher query will be returned and you will be able to select and view each one.
+
+<TODO: Add screenshot here>
+
+* When using code blocks in rich collection descriptions, you can specify one of the following languages to have
+  syntax highlighting enabled: `cypher`, `bash`, `python`, `css`, `c`, `cpp`, `json`, `sql`, and `yaml`. Check the
+  examples for syntax highlighting on [Markdown Guide](https://www.markdownguide.org/extended-syntax/#syntax-highlighting).
+
+<TODO: Add screenshot here>
+
+* Query modules are now sorted alphabetically which will enable easier and faster. A search has been added for query
+  module procedures if there are more than 5 procedures within a module for easier narrowing down (e.g. `nxalg`
+  with [49 procedures](https://memgraph.com/docs/mage/query-modules/python/nxalg)).
+* Rendering graph view for medium/larger graphs might take time, so for any graph with more than 5k nodes or 10k edges
+  you will be asked if you want to proceed with the graph visualization or switch to the data view. This also fixed
+  showing a default graph view for each new query execution and accidentally rendering the full database graph.
+* Changes in the Cypher query editor and GSS style editor will be automatically saved in rich collections. You will
+  also have the option to do the manual save like before.
```

Comment:
* In rich collections, you can now choose between saving changes in the Cypher query editor and GSS style editor manually or automatically.

* Besides manually saving changes in the Cypher query and GSS style editor, they will also be saved automatically.

I'm again not sure what the feature is... if they are saved automatically, why would I save them manually as well? Or do I choose between manual save and autosave?

Reply:
yes, perfect! :)